converter/avcodecs.py

```diff
@@ -716,6 +716,18 @@ class NVEncH264(H264Codec):
     codec_name = 'nvenc_h264'
     ffmpeg_codec_name = 'nvenc_h264'
 
+class H264VAAPI(H264Codec):
+    """
+    H.264/AVC video codec.
+    """
+    codec_name = 'h264vaapi'
+    ffmpeg_codec_name = 'h264_vaapi'
+
+    def _codec_specific_produce_ffmpeg_list(self, safe, stream=0):
+        optlist = []
+        optlist.extend(['-vaapi_device', '/dev/dri/renderD128'])
+        optlist.extend(['-vf', "format=nv12,hwupload"])
+        return optlist
 
```

Comment:
I made a similar mode but in [mkvtomp4](https://github.com/nzbjamz/sickbeard_mp4_automator/commit/8a91378aeadf4f56bc49557b41c7ed2d596c5e33#diff-34b703ed85dfe4e4a819306499832c2c)